pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -173,6 +173,25 @@ def method_date(self):
         help="Many 2 Many"
     )
 
+    def _compute_name(self):
```

Comment:
How this method is identified as computed if there is not defined from `meth(compute='compute_method')`?

Reply:
The computed method `_compute_name`  is defined in

https://github.com/OCA/pylint-odoo/blob/0a6242ba4ad720a867e8e23953732aa568c97329/pylint_odoo/test_repo/broken_module/models/broken_model.py#L28-L35

And the method `my_method_compute` is defined in

https://github.com/OCA/pylint-odoo/blob/0a6242ba4ad720a867e8e23953732aa568c97329/pylint_odoo/test_repo/broken_module/models/broken_model.py#L38-L44

I just use the existing code to avoid created another code to test this funcionality